lib/modules/touch.js

```diff
@@ -0,0 +1,50 @@
+import { contentLoaded } from '../utils';
+
+export default (shower) => {
+    contentLoaded(() => {
+        let action = false;
+        let clickable = false;
+
+        document.addEventListener('touchstart', (ev) => {
```

Comment:
Whenever possible, we try to make Shower embeddable by not making an assumption that it's the only element on the page. Can we use `shower.container.addEventListener` here?

Reply:
Also nit: `ev` => `event`, we don't shorten words.